src/vmm/src/builder.rs

```diff
@@ -28,6 +29,7 @@ use kbs_types::Tee;
 use crate::device_manager;
 #[cfg(feature = "tee")]
 use crate::resources::TeeConfig;
+use crate::signal_handler::register_sigint_handler;
```

Comment:
Please put the `register_sigint_handler` import and its uses behind a `#[cfg(target_os = "linux")]` build-time conditional.

Reply:
Oh the whole signal_handler file is not compiled on macOS. 

Btw we can `cargo clippy --target aarch64-apple-darwin` to the pipeline, which checks agains simple issues like this (non existing imports), but this currently generates some warnings, so maybe it should could be in another PR.

Reply:
Yeah, let's get this one merged first. I'll prepare a new PR setting up the CI for aarch64-apple-darwin.